app/views/admin/products_v3/_variant_row.html.haml

```diff
@@ -18,7 +18,7 @@
       = f.hidden_field :unit_value
       = f.hidden_field :unit_description
       = f.text_field :unit_value_with_description,
-          value: unit_value_with_description(variant), 'aria-label': t('admin.products_page.columns.unit_value'), required: true
+          value: unit_value_with_description(variant), 'aria-label': t('admin.products_page.columns.unit_value')
```

Comment:
Having `required` here is causing an error upon saving in the following scenario:
- **New varaint** button is clicked to create a new variant
- As per Old UI behavior, unit_value is empty for this product.
- However, in the new UI, if this variant is changed then the saving form doesn't work because this field is hidden on the UI until its pop-up is opened
- JS tries to find it to show the required validation error but gets broken because it's not able to find it on the UI
- Hence neither the required error gets displayed nor the form gets submitted.
- This PR removes the required validation which won't cause the failure of the above use-case however, the backend is [handling this blank field error](https://github.com/openfoodfoundation/openfoodnetwork/pull/12710/files#diff-d8fe63147fa63b4c9fb35f7c510dd1c1990c61855be8420f1ce7ebf78967e2ddR517), so it will return the erorr "_can't be blank"_ which will be displayed below it.

Reply:
> JS tries to find it to show the required validation error but gets broken because it's not able to find it on the UI

This validation is handled by the browser, probably in Javascript. I think that's a fair trade off, I am wondering if we could show the pop up with error on form submit so we can still benefit from the browser validation but it's probably not worth the effort.

Reply:
FYI: I ran into the same issue while working on the Product Refactor and I managed to implement the solution I was talking about above. It ended up pretty simple in the end.

Reply:
Thanks for implementing this one. ♥️ 
I tried to implement the above, maybe I was looking at it from a different POV and was facing issues. Looking forward to your solution to this in the refactor 😄

Reply:
The `required` JS validation was added after discussing with Mario. It ensures that the popout can't be closed if it's empty, which helps avoid the problem being hidden.
But as you've found, we have a problem if the field is empty and the popout never gets opened.

This change seems like the most practical best solution 👍  
I'm not sure if the red border will show on the popout, until you open it. But if there error text is there, I think that's ok.